_squirtle/mod.ts

```diff
@@ -70,29 +67,82 @@ const runLogin = async () => {
   await runUser(null, user, env)
 }
 
+const browse = (location: string): Promise<Deno.ProcessStatus> => {
+  return new Promise<Deno.ProcessStatus>((resolve, reject) => {
+    switch (process.platform) {
+      case 'aix':
+      case 'freebsd':
+      case 'linux':
+      case 'openbsd':
+        return Deno.run({
+          cmd: ['gp', 'preview', '--external', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .catch(() =>
+            Deno.run({
+              cmd: ['xdg-open', location],
+              stderr: 'null',
+              stdout: 'null',
+            })
+              .status()
+          )
+          .then(resolve)
+          .catch(reject)
+      case 'darwin':
+        return Deno.run({
+          cmd: ['open', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+      case 'win32':
+        return Deno.run({
+          cmd: ['cmd', '/c', 'start', location],
+          stderr: 'null',
+          stdout: 'null',
+        })
+          .status()
+          .then(resolve)
+          .catch(reject)
+    }
+  })
+    .then((status: Deno.ProcessStatus) => {
+      if (status.success)
+        console.log("opened %s", location)
+      return status
+    })
+}
+
 const runLogout = async (
-  _: any, 
+  _: any,
   internal = false
 ) => {
-  const run1 = Deno.run({
-    cmd: ['gp', 'env', '-u', 'AUTH_TOKEN'],
+  const { AUTH_TOKEN, ENV } = await getEnv()
+  if (!AUTH_TOKEN)
+    return console.log(`Please run the <login> command first`);
```

Comment:
?? Please login before you logout? Let's just skip this and assume someone is already logged out.

Reply:
it was there before - did not remove it. but it makes perfect sense to do so